package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "scip",
   "version": "0.1.0",
-  # For production builds, homepage must be changed to the absolute URL of the portal
-  "homepage": "http://localhost/frontend",
+  "homepage": ".",
```

Comment:
Great. 

And you'll need to set the env var `PUBLIC_URL` to the appropriate URL (for local development, the URL proxied by nginx; for server deployments, the HAProxy public URLs) so that it knows where it is. I think that info should go into section "Setting up the frontend" along with all the other env vars.